src/test/kotlin/uk/gov/justice/digital/hmpps/prisonregister/ApplicationIntegrationTest.kt

```diff
@@ -26,13 +24,20 @@ class ApplicationIntegrationTest(@Autowired val jwtAuthHelper: JwtAuthHelper) :
       .get()
```

Comment:
I can't see a test in `OMU email address life-cycle` that returns `Forbidden` - where the role `MAINTAIN_REF_DATA` is not present.  Possibly the same below in `VCC email address life-cycle`